occurrences/tests/snapshots/snap_test_api.py

```diff
@@ -38,7 +39,7 @@
             "name": "Increase thank certainly again thought summer. Beyond than trial western.",
             "occurrences": {"edges": []},
             "person": {"name": "William Brewer"},
-            "studyLevel": "GRADE_6",
```

Comment:
Here is something wrong. Grade 6 should be list in a new snapshot if it was in previous.

Reply:
Seems like this was just some random generated data that is not important here.